setup.py

```diff
@@ -3,7 +3,7 @@
 import os
 import setuptools
 
-from src.drozer import meta
+from drozer import meta
```

Comment:
Any reason why this has been changed? This was changed to src.drozer so you no longer need to set environment variables so it negated a lot of the bug reports.